blogvault/wpcom-migration
# Add static migration progress screen

Related to https://github.com/Automattic/dotcom-forge/issues/8727

# Screenshots

Desktop:

<img width="1188" alt="Screenshot 2024-08-20 at 2 04 21 PM" src="https://github.com/user-attachments/assets/3a658690-8dc9-485a-b5d0-13c920ae8bc7">

<img width="1184" alt="Screenshot 2024-08-20 at 3 43 46 PM" src="https://github.com/user-attachments/assets/3c0a2305-5647-4b80-939b-07633a5bef4c">

Mobile: TBD

# Testing Instructions

1. Apply this branch locally or download it as a .zip. You could use [jurassic.ninja](https://jurassic.ninja/) for a quick environment setup.
2. Go to `/wp-admin/admin.php?page=automattic&static=migration-progress`
3. Compare the page to the [design](https://www.figma.com/design/c9jLiGBEqrxgEpHc8ccVFr/WP.com-migration-plugin-screen-designs?node-id=163-569&t=a1RJbTkLogyK90p0-0). Having small differences should be fine, IMO.
4. Make sure it looks good on mobile and the different browsers.
5. To enable the modal for testing, update line 171 of style.css to `display: block`
